guacamole/src/main/webapp/translations/fr.json

```diff
@@ -624,7 +624,9 @@
         
         "FORMAT_STARTDATE" : "@:APP.FORMAT_DATE_TIME_PRECISE",
 
-        "HELP_SESSIONS" : "Toutes les connexions actives Guacamole sont listées ici. Si vous souhaitez en fermer une ou plusieurs, sélectionner les et cliquer sur \"Fermer Sessions\". La fermeture d'une session déconnectera immédiatement l'utilisateur.", 
+        "HELP_SESSIONS" : "
+137/5000
```

Comment:
What happened here?

Reply:
Copypasta.  From Google Translate.  Will fix when I fix the wording.